### ISSUE TITLE
Add the uplift repositories to code-review-bot

### ISSUE THREAD

**Issue by globau** [mozilla/code-review#2004]
Please configure code-review-bot to process revisions from the listed Phabricator repositories below.  Each is related to uplifts, and should be handled the same as mozilla-central:

- mozilla-beta
- mozilla-release
- mozilla-esr115

Thanks!

**Comment by globau** [comment]
Blocks https://bugzilla.mozilla.org/show_bug.cgi?id=1861780

**Comment by marco-c** [comment]
I added Beta, we should confirm it's working and then I will add the others.

**Comment by cgsheeh** [comment]
I took a look at some of the beta uplifts and noticed a few things. In some cases like [D199896](https://phabricator.services.mozilla.com/D199896) we see the patch fail to apply as expected, confirmed by attempting to apply the patch manually.

In other cases like [D199918](https://phabricator.services.mozilla.com/B621243) the patch fails to apply because "the parent revision does not exist on mozilla-unified". I'm not sure where the parent revision is being pulled from, but in the case of uplift repositories the patches should always be applied to the latest commit from that branch (ie the `beta` firefoxtree/bookmark in mozilla-unified). We are setting the `sourceControlBaseRevision` property on the Phabricator diff to the latest commit from the repository, pulled from the `attachments.metrics.recentCommit.identifier` property on the Phabricator `diffusion.repository.search` endpoint response.

**Comment by marco-c** [comment]
We are reading the `refs.base.identifier` property from the `differential.diff.search` endpoint response: https://github.com/mozilla/libmozdata/blob/50135a6fb76f47b7086516df59c3d55a5baaa9a3/libmozdata/phabricator.py#L265.

**Comment by marco-c** [comment]
I'll close this as it has been done, let's track the other problem in another issue (we can either always assume latest from branch, or use the other property if `refs.base.identifier` is not the right one).

**Comment by cgsheeh** [comment]
> I added Beta, we should confirm it's working and then I will add the others.

We still need to add the other uplift repos, according to this comment.

**Comment by marco-c** [comment]
I added them yesterday, sorry I forgot to add a comment here.

**Comment by globau** [comment]
I've added release and esr to Phab's herald rule, closing [bug 1861780](https://bugzilla.mozilla.org/show_bug.cgi?id=1861780)

Sheehan's filed #2057 for the parent commit issue.

**Comment by marco-c** [comment]
We are running out of disk space and I couldn't find a way to increase it on Heroku.
Reopening this as I had to remove mozilla-esr115 and mozilla-release.

**Comment by marco-c** [comment]
This is what I removed from the config, so we can copy it back once we find a way to increase the disk size or once we move off of Heroku (https://mozilla-hub.atlassian.net/browse/OPST-766):
```
    - checkout: robust
      try_url: ssh://hg.mozilla.org/try
      try_mode: json
      try_name: try
      name: mozilla-release
      ssh_user: reviewbot
      url: https://hg.mozilla.org/releases/mozilla-release
      decision_env_revision: GECKO_HEAD_REV
      decision_env_repository: GECKO_HEAD_REPOSITORY
      decision_env_prefix: GECKO
      use_latest_revision: true
    - checkout: robust
      try_url: ssh://hg.mozilla.org/try
      try_mode: json
      try_name: try
      name: mozilla-esr115
      ssh_user: reviewbot
      url: https://hg.mozilla.org/releases/mozilla-esr115
      decision_env_revision: GECKO_HEAD_REV
      decision_env_repository: GECKO_HEAD_REPOSITORY
      decision_env_prefix: GECKO
      use_latest_revision: true
```

**Comment by globau** [comment]
ohno; this work of code-review-bot is an important part of the uplift process.
Do you have a time frame for when you expect more disk to be added?

**Comment by marco-c** [comment]
I can't find a way to increase the local disk size on Heroku, other than moving from a performance-m dyno to a standard-2x one. Unfortunately, standard-2x has too little memory and runs out during cloning.

I can't think of any other options than https://mozilla-hub.atlassian.net/browse/OPST-766.

**Comment by marco-c** [comment]
Depends on #2194.

**Comment by marco-c** [comment]
@Archaeopteryx we should be able to do this (and #2204) now that the `events` service no longer exists.

**Comment by Archaeopteryx** [comment]
mozilla-release and mozilla-esr128 have been added to code-review's configuration.
